src/control/LatexController.cpp

```diff
@@ -63,84 +36,21 @@ LatexController::~LatexController() { this->control = nullptr; }
  * Find the tex executable, return false if not found
  */
 auto LatexController::findTexDependencies() -> LatexController::FindDependencyStatus {
-    gchar* pdflatex = g_find_program_in_path("pdflatex");
-    if (!pdflatex) {
-        string msg =
-                _("Could not find pdflatex in PATH.\nPlease install pdflatex first and make sure it's in the PATH.");
-        return LatexController::FindDependencyStatus(false, msg);
-    }
-    this->pdflatexPath = pdflatex;
-    g_free(pdflatex);
-
-    // Check for 'standalone' latex package
-    static gchar* kpsewhichArgs[] = {g_strdup("kpsewhich"), g_strdup("standalone"), nullptr};
-    auto kpsewhichFlags = GSpawnFlags(G_SPAWN_DEFAULT | G_SPAWN_SEARCH_PATH | G_SPAWN_STDOUT_TO_DEV_NULL);
-    GError* kpsewhichErr = nullptr;
-    gint kpsewhichStatus = 0;
-    g_spawn_sync(nullptr, kpsewhichArgs, nullptr, kpsewhichFlags, nullptr, nullptr, nullptr, nullptr, &kpsewhichStatus,
-                 &kpsewhichErr);
-    if (kpsewhichErr != nullptr) {
-        string msg =
-                FS(_F("Error: {1}\nCould not find kpsewhich in PATH; please install kpsewhich and put it on path.") %
-                   kpsewhichErr->message);
-        g_error_free(kpsewhichErr);
+    std::ifstream is(this->settings.globalTemplatePath.string(), std::ios_base::binary);
+    if (!is.is_open()) {
+        g_message("%s", this->settings.globalTemplatePath.string().c_str());
+        string msg = _("Global template file does not exist. Please check your settings.");
         return LatexController::FindDependencyStatus(false, msg);
     }
-    if (kpsewhichStatus != 0) {
-        string msg = FS(_F("Could not find the LaTeX package 'standalone'.\nPlease install standalone (found in "
-                           "texlive-latex-extra) and make sure "
-                           "it's accessible by your LaTeX installation."));
+    this->latexTemplate = std::string(std::istreambuf_iterator<char>(is), {});
+    if (!is.good()) {
+        string msg = _("Failed to read global template file. Please check your settings.");
         return LatexController::FindDependencyStatus(false, msg);
     }
```

Comment:
Here (as indicated by @Technius ) a check is needed if the template path name is a regular file (not a folder, for instance).